modules/perception/onboard/component/trafficlights_perception_component.cc

```diff
@@ -39,6 +40,9 @@
 namespace apollo {
 namespace perception {
 namespace onboard {
+
+using ::apollo::cyber::Clock;
+
 using TLCamID = apollo::perception::TrafficLightDetection::CameraID;
 using apollo::cyber::common::GetAbsolutePath;
```

Comment:
Better to put `using apollo::cyber::Clock;` here.

Reply:
OK

Reply:
Actually what I mean is put `using apollo::cyber::Clock` on the next line of `using::apollo::cyber::common::GetAbsolutePath`...

Reply:
get it